services/big_panda.rb

```diff
@@ -42,6 +46,10 @@ def body
     body['runbook_url'] = payload['alert']['runbook_url'] if present?(payload['alert']['runbook_url'])
     body['link'] = alert_link(payload['alert']['id'])
 
+    if payload[:triggered_by_user_test]
+      body['note'] = "This is a test message sent by #{payload[:auth][:email]} via metrics.librato.com/alerts. No action is required."
```

Comment:
It seems like we're duplicating this test message for each service. Maybe we should have a helper method

```
def render_test_message(email)
   "This is a test.... #{email} ...."
end
```
That we can call from each service?